src/Command/BindDeputiesToDistrictsCommand.php

```diff
@@ -83,31 +83,48 @@ protected function execute(InputInterface $input, OutputInterface $output)
     private function bindDeputies(array $deputies): void
     {
         foreach ($deputies as $deputy) {
-            $adherent = $this->em->getRepository(Adherent::class)->findOneBy(['uuid' => $deputy['uuid']]);
-            if (!$adherent) {
-                $this->io->error(sprintf("Adherent with uuid '%s' does not exist.", $deputy['uuid']));
-                ++$this->nbErrors;
-                continue;
+            if (!$deputy['code_dpt'] || !$deputy['email']) {
+                return;
+            }
+
+            if ('FDE' === $deputy['code_dpt']) {
+                $params = [
+                    'code' => sprintf('FDE-%02d', $deputy['num_circo']),
+                ];
+            } elseif (1 === \strlen($deputy['code_dpt'])) {
+                $params = [
+                    'departmentCode' => sprintf('%02d', $deputy['code_dpt']),
+                    'number' => (int) $deputy['num_circo'],
+                ];
+            } else {
+                $params = [
+                    'departmentCode' => $deputy['code_dpt'],
+                    'number' => (int) $deputy['num_circo'],
+                ];
             }
 
-            $district = $this->em->getRepository(District::class)->findOneBy([
-                'departmentCode' => sprintf('%05d', $deputy['code_dpt']),
-                'number' => (int) $deputy['num_circo'],
-            ]);
+            $district = $this->em->getRepository(District::class)->findOneBy($params);
```

Comment:
it's OK for this command one shot, but try to avoid to do the same things into the loop, like `$this->em->getRepository(District::class)` action